src/common.h

```diff
@@ -102,8 +102,8 @@ enum class Result {
   Error,
 };
 
-#define WABT_SUCCEEDED(x) ((x) == Result::Ok)
-#define WABT_FAILED(x) ((x) == Result::Error)
+#define WABT_SUCCEEDED(x) ((x) == ::wabt::Result::Ok)
+#define WABT_FAILED(x) ((x) == ::wabt::Result::Error)
```

Comment:
Really?  wabt::Result::Ok existing in two places is surprising.   But I'm prepared to believe it :)